### PR TITLE
Register Suppressor [https://github.com/Ismael-VC/Suppressor.jl.git]:…

### PR DESCRIPTION
**Pull request by Ismael-VC**
… v0.0.1, v0.0.2

### CODE REVIEW

File: Suppressor/versions/0.0.2/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.4
+Compat
```
Comment (tkelman) [inline Suppressor/versions/0.0.2/requires]: should really have a minimum version requirement here that implements the features you need

**Comment by tkelman** [comment]
Dunno if this was intended or an accident, but the 0.0.1 and 0.0.2 tags here point to the same commit.

**Comment by tkelman** [comment]
Should also write some tests and enable CI. Some methods of redirecting output are very susceptible to freezing on Windows for example. Not sure whether that includes what you're doing here, but you should test it.

**Comment by Ismael-VC** [comment]
> Dunno if this was intended or an accident, but the 0.0.1 and 0.0.2 tags here point to the same commit.

I'm having trouble finding a way to test this, but I'll do it.

The double tagging the same commit was an unintended mistake, do you know how could I undo the last tag, I've not found any reference on that.

**Comment by Ismael-VC** [comment]
superseded by https://github.com/JuliaLang/METADATA.jl/pull/6659

**Pull request closed without merging**